### PR TITLE
ola: Update to 0.10.7 and switch to codeload

### PR DESCRIPTION
**Pull request by neheb**
Simpler and easier to bump version.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @bk138 
Compile tested: ipq806x

### CODE REVIEW

File: net/ola/Makefile [hunk 1]
```diff
@@ -9,20 +9,18 @@
...
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/OpenLightingProject/ola/tar.gz/$(PKG_VERSION)?
+PKG_HASH:=95b5e99775f700a9cdf6e2219eccfc8c037fa8841f87cb00344359c4b8c7bf26
+
 PKG_LICENSE:=LGPL-2.1+
 
 PKG_FIXUP:=libtool
-PKG_INSTALL:=1
 
+PKG_INSTALL:=1
```
Comment (bk138) [inline net/ola/Makefile]: why those ordering changes?

File: net/ola/patches/002-remove-visibility-inlines-hidden-flag.patch [hunk 2]
```diff
@@ -2,7 +2,7 @@ diff --git a/Makefile.am b/Makefile.am
 index 22647be..7dcf0de 100644
 --- a/Makefile.am
 +++ b/Makefile.am
-@@ -32,7 +32,7 @@ AM_DISTCHECK_CONFIGURE_FLAGS = --enable-python-libs
+@@ -45,7 +45,7 @@ AM_DISTCHECK_CONFIGURE_FLAGS = --enable-python-libs
```
Comment (bk138) [inline net/ola/patches/002-remove-visibility-inlines-hidden-flag.patch]: same here. what was changed?

Comment (neheb) [reply net/ola/patches/002-remove-visibility-inlines-hidden-flag.patch]: Patch was refreshed.

File: net/ola/Makefile (hunk 1 above)
Comment (neheb) [reply net/ola/Makefile]: Consistency with other packages.

Comment (bk138) [reply net/ola/Makefile]: ok

File: net/ola/patches/002-remove-visibility-inlines-hidden-flag.patch (hunk 2 above)
Comment (bk138) [reply net/ola/patches/002-remove-visibility-inlines-hidden-flag.patch]: Did you force-push the branch? I still see the 'old' diff.

Comment (neheb) [reply net/ola/patches/002-remove-visibility-inlines-hidden-flag.patch]: The only change was the line numbers.

**Comment by dibdot** [comment]
@bk138 ready to merge?

**Comment by bk138** [comment]
Yeah

**Comment by cshoredaniel** [comment]
Please note that project stance on codeload is under review: https://github.com/openwrt/packages/issues/6748

**Pull request merged**